### PR TITLE
Enable cookies

### PR DESCRIPTION
**Pull request by echanlee**

### CODE REVIEW

File: src/SwipeProfiles.js [hunk 1]
```diff
@@ -137,64 +137,77 @@ class SwipeProfiles extends React.Component {
   }
 
   render() {
+    
+    const id = this.state.id;
     const potentialFriends = this.state.potentialFriends;
     const displayedUserId = this.state.displayedUserId;
     const error = this.state.error;
+
```
Comment (wclui) [inline src/SwipeProfiles.js]: is there any way to save this somewhere and reuse it since it's on almost every page?

**Review by wclui — CHANGES_REQUESTED** [review]

File: src/SwipeProfiles.js (hunk 1 above)
Comment (echanlee) [reply src/SwipeProfiles.js]: It's saved in the cookies. When it sets the state initially is calls the getCookies method to retrieve the value. So it's saved in the cookies so we can use it on every page

Comment (wclui) [reply src/SwipeProfiles.js]: Do you mean line 146 to 152 has to physically be on all the pages? can we not export these lines of code and reuse it?

**Review by wclui — APPROVED** [review]

File: src/SwipeProfiles.js (hunk 1 above)
Comment (echanlee) [reply src/SwipeProfiles.js]: Oh I thought you meant the id value, okay I can create a method that does that

**Pull request merged**